src/com/sun/javatest/Script.java

```diff
@@ -208,6 +211,10 @@ public abstract class Script {
     private boolean jtrIfPassed =
             System.getProperty("javatest.script.jtrIfPassed", "true").equals("true");
 
+    /**
+     * tries to search for a statusTransformer, no status transformations will be done if the value is null
```

Comment:
Technically speaking, the field does not "try to search"

I would suggest something like

```suggestion
     * Contains {@link StatusTransformer} that transforms the test result status, or {@code null} if no transformer service was found.
```

Reply:
sure, sorry comments and naming are not my strong suite